app/views/spina/admin/pages/_form.html.erb

```diff
@@ -4,67 +4,36 @@
         <%= heroicon("external-link", style: :solid, class: "w-4 h-4") %>
       <% end %>
     <% end %>
-    
     <% header.actions do %>
       <%= render Spina::Pages::ActionsComponent.new(page: @page, locale: @locale) %>
     <% end %>
-    
     <% header.navigation do %>
       <nav class="-mb-3 mt-4">
         <ul class="inline-flex w-auto rounded-md bg-white">
-          <button type="button" class="block px-3 leading-relaxed py-1 hover:text-gray-800 rounded-md text-gray-400 font-medium text-sm flex items-center whitespace-nowrap" data-action="tabs#show" data-tabs-target="button" data-pane-id="page_content">
-            <%= heroicon('document-text', class: 'w-4 h-4 mr-1 -ml-1 opacity-75') %>
-            <%=t 'spina.pages.page_content' %>
-          </button>
-          <button type="button" class="block px-3 leading-relaxed py-1 rounded-md hover:text-gray-800 text-gray-400 font-medium text-sm flex items-center whitespace-nowrap" data-action="tabs#show" data-tabs-target="button" data-pane-id="seo">
-            <%= heroicon('search', style: :solid, class: 'w-4 h-4 mr-1 -ml-1 opacity-75') %>
-            <span class="hidden md:inline">
-              <%=t 'spina.pages.search_engines' %>
-            </span>
-            <span class="md:hidden">
-              SEO
-            </span>
-          </button>
-          <button type="button" class="block px-3 leading-relaxed py-1 rounded-md hover:text-gray-800 text-gray-400 font-medium text-sm flex items-center whitespace-nowrap" data-action="tabs#show" data-tabs-target="button" data-pane-id="advanced">
-            <%= heroicon('cog', style: :solid, class: 'w-4 h-4 mr-1 -ml-1 opacity-75') %>
-            <%=t 'spina.pages.advanced' %>
-          </button>
+          <% @tabs.each do |tab_name| %>
+            <%= render Spina::Pages::TabButtonComponent.new(
+              page: @page,
+              tab_name: tab_name,
+            ) %>
+          <% end %>
         </ul>
       </nav>
     <% end %>
   <% end %>
-  
   <div class="p-4 md:p-8">
     <%= form_with model: @page, url: spina.admin_page_path(@page), id: dom_id(@page) do |f| %>
       <%= hidden_field_tag :locale, @locale %>
-      
       <% Mobility.with_locale(@locale) do %>
-        <div data-tabs-target="pane" id="page_content">
-          <div class="max-w-5xl">
-            <label class="font-medium text-gray-700">
-              <%= Spina::Page.human_attribute_name(:title) %>
-            </label>
-          
-            <%= render Spina::Forms::TextFieldComponent.new(f, :title, size: "lg") %>
-            
-            <turbo-frame id="page_content" src="<%= spina.edit_content_admin_page_path(f.object, locale: @locale) %>">
-            </turbo-frame>
+        <% @tabs.each do |tab_name| %>
+          <div data-tabs-target="pane" id="<%= tab_name %>" <%= 'hidden' unless tab_name == @tabs.first %>>
```

Comment:
Very minor nitpick, but you could also write this as:

```erb
<% @tabs.each.with_index do |tab_name, index| %>
  <div data-tabs-target="pane" id="<%= tab_name %>" <%= "hidden" unless index.zero? %>>
```

Reply:
Sounds good, changed!